static-assets/components/cstudio-forms/controls/rte.js

```diff
@@ -429,6 +436,8 @@ CStudioAuthoring.Module.requireModule(
 
             style_formats_merge: styleFormatsMerge,
 
+            paste_word_valid_elements: rtePasteWordElements,
```

Comment:
Per [tinymce docs](https://www.tiny.cloud/docs/plugins/opensource/paste/#paste_word_valid_elements), "this property is deprecated and will be removed in v6". You are using this for **your** plugin, correct? Not relying on tinymce's paste plugin?

For tinymce's plugin, `paste_word_valid_elements` needs to be `false`. Do you need to do something of the sorts?

Reply:
Hi Roy, yes, we are using this for our own plugin. We don't rely on the default `paste` plugin. But our plugin has some logic borrowed from `paste`.  As of checking with @russdanner, we want to have an option to update valid elements which should be kept while pasting a Word document. Only valid elements are kept.